src/3d/qgscameracontroller.h

```diff
@@ -217,11 +217,10 @@ class _3D_EXPORT QgsCameraController : public QObject
     void zoomCameraAroundPivot( const QVector3D &oldCameraPosition, double zoomFactor, const QVector3D &pivotPoint );
 
     /**
-     * Returns TRUE if the camera controller will handle the specified key \a event,
-     * preventing it from being instead handled by parents of the 3D window before
-     * the controller ever receives it.
+     * If the event is relevant, handles the event and returns true, otherwise false.
```

Comment:
These should be formatted like:

```suggestion
     * If the event is relevant, handles the event and returns TRUE, otherwise FALSE.
```

So that sipify/doxygen correctly format them as c++/python literals

Reply:
Thanks, done.